contribs/gmf/apps/desktop/contextualdata.html

```diff
@@ -4,15 +4,15 @@
       Swiss grid (LV03)
     </td>
     <td>
-      {{coord_21781|ngeoNumberCoordinates}}
+      {{coord_21781|ngeoNumberCoordinates:0:'{x} / {y}'}}
```

Comment:
Finally we want a /?

Reply:
The problem with the comma is that in english we will have: `631,285, 152,450` which is not very readable.

Reply:
And what about making it translatable?